orchagent/intfsorch.cpp

```diff
@@ -155,6 +155,7 @@ bool IntfsOrch::setIntf(const string& alias, sai_object_id_t vrf_id, const IpPre
         return false;
     }
 
+    vrf_id = port.m_vr_id;
```

Comment:
Can you explain why is this required?

Reply:
But this vrf_id in this same function is to `addRouterIntfs(vrf_id, port)`
How does it work for rif and not for ip2me route?

Reply:
`addRouterIntfs(vrf_id, port)` is a different flow (called from `VnetOrch`), `ip2Me ` route is not from `VnetOrch`. Remember, the IP address config doesn't have Vnetname in it.